src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -462,6 +452,18 @@ private static void ValidateBase64EncodedSignature(string base64EncodedSignature
         }
     }
 
+    private const int checksumSizeInBytes = sizeof(uint);
+
+    public static bool ValidateChecksum(string key, ulong checksumSeed, out byte[] bytes)
```

Comment:
>ValidateChecksum [](http://example.com/codeflow?start=23&length=16)

TryValidateBase64 is a very heavyweight API, intended to support comprehensive testing of the emerging HIS v1 format (when it was, in fact, newly introduced).